R/createInputForDescriptives.r

```diff
@@ -194,7 +202,7 @@ createInputForDescriptives.GADSdat <- function ( GADSdat, idExpr = "^ID", impExp
     ### nach Laufnummer sortieren und dann die Laufnummer- und Netzwerk-Spalte entfernen
            vari <- data.frame(vari[sort(vari[,"laufnummer"],decreasing=FALSE,index.return=TRUE)$ix,-match(c("laufnummer","nw"), colnames(vari))])
     ### consistency checks
-           variV<- vari[which(vari[,"type"] == "variable"),]
+           variV<- vari[which(vari[,"type"] %in% c("variable", "item", "fake_item")),]

```

Comment:
LGTM!

Nit: Should `"variable"` be included here or should it be omitted? Are there any instances, in which such a check is meaningful?

Reply:
`variable` should be included for the following check which applies in the same way for `variable`, `item` and `fake_item`.